test/cql-pytest/run.py

```diff
@@ -216,6 +216,7 @@ def run_scylla_cmd(pid, dir):
         # Set up authentication in order to allow testing this module
         # and other modules dependent on it: e.g. service levels
         '--authenticator', 'PasswordAuthenticator',
+        '--strict-allow-filtering', 'true',
```

Comment:
Adding an option that changes the default behavior in run.py is a bit of a problem, because it means that the test can *only* be run with run.py, and you can't run Scylla yourself (if you forget this non-default option) and run tests against it. But then again, this is exactly why I was against this flag (that @avikivity requested) in the first place - it means that this patch by default does not fix the bug that we had, and you need this "--strict-allow-filtering=true" option to fix it, so even if we unfortunately don't make this option the default, it does make sense that our tests run with it, otherwise they will encounter a bug we already fixed...

Reply:
For users, their test is their workload. They must prefer a false negative (the application keeps working despite a missing ALLOW FILTERING) to a true positive (the application crashes due to missing ALLOW FILTERING).

Reply:
@avikivity I just ran into this option in test/cql-pytest/run.py again, and I think it was a mistake. NOBODY will ever use this option manually (who will even hear about it?) so it's pointless to test how Scylla behaves with this option which nobody uses :-(